fpmsyncd/routesync.cpp

```diff
@@ -102,10 +101,9 @@ void RouteSync::parseEncap(struct rtattr *tb, uint32_t &encap_value, string &rma
     parseRtAttrNested(tb_encap, 3, tb);
     encap_value = *(uint32_t *)RTA_DATA(tb_encap[VXLAN_VNI]);
     memcpy(&mac_buf, RTA_DATA(tb_encap[VXLAN_RMAC]), MAX_ADDR_SIZE);
-    vlan = *(uint32_t *)RTA_DATA(tb_encap[VXLAN_VLAN]);
```

Comment:
How do we get the vlan data if it is not nested? Do we need to handle that?

Reply:
Vlan information was used to create the nexthop,  Similar to normal route(no evpn routes) nexthop information is extracted from tb[RTA_OIF] attribute.